plugins/misc/w_helicity.h

```diff
@@ -0,0 +1,142 @@
+// -*- C++ -*-
+// author: Andre Zimermmane Santos
+// short: for computing W-helicity angle in the ttbar system
+
+#ifndef FWK_W_HELICITY_H
+
```

Comment:
container_util.h provides index_with_key, so it needs to be included.

Reply:
updated